src/components/cylc/gscan/GScan.vue

```diff
@@ -19,8 +19,87 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <div
     class="c-gscan"
   >
+    <!-- filters -->
     <div
-      v-if="workflows && workflows.length > 0"
+      class="d-flex flex-row mx-4 mb-2"
+    >
+      <v-text-field
+        v-model="searchWorkflows"
+        clearable
+        flat
+        dense
+        hide-details
+        outlined
+        placeholder="Search"
+        class="flex-grow-1 flex-column"
+      />
+      <v-menu
+        v-model="showFilterTooltip"
+        :close-on-content-click="false"
+        offset-x
+      >
+        <!-- button to activate the filters tooltip -->
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn
+            v-bind="attrs"
+            v-on="on"
+            link
+            icon
+            class="flex-grow-0 flex-column"
+            @click="showFilterTooltip = !showFilterTooltip"
+          >
+            <v-icon>{{ svgPaths.filter }}</v-icon>
+          </v-btn>
+        </template>
+        <!-- filters tooltip -->
+        <v-card
+          max-height="250px"
+        >
+          <v-list
+            dense
+          >
+            <div
+              v-for="filter in filters"
+              :key="filter.title"
+            >
+              <v-list-item dense>
+                <v-list-item-content ma-0>
+                  <v-list-item-title>
+                    <v-checkbox
+                      :label="filter.title"
+                      :input-value="allItemsSelected(filter.items)"
+                      value
+                      dense
+                      hide-details
+                      hint="Toggle all"
+                      @click="toggleItemsValues(filter.items)"
+                    ></v-checkbox>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+              <v-divider />
+              <v-list-item
+                v-for="item in filter.items"
+                :key="`${filter.title}-${item.text}`"
+                dense
+              >
+                <v-list-item-action>
+                  <v-checkbox
+                    :label="item.text"
+                    v-model="item.model"
+                    dense
+                    height="20"
+                  ></v-checkbox>
+                </v-list-item-action>
+              </v-list-item>
+            </div>
+          </v-list>
+        </v-card>
+      </v-menu>
+    </div>
```

Comment:
These filters are not looking really nice here and in the Tree component or view. Have added a post-it to take a better look, see if we can unify or find a better way to organize them. If so, will create a follow-up issue :+1: